src/Compilers/Core/Portable/FileSystem/FileUtilities.cs

```diff
@@ -126,7 +130,7 @@ private static string ResolveRelativePath(PathKind kind, string path, string bas
                         return null;
                     }
 
-                    if (path.Length == 1)
+                    if (path!.Length == 1)
```

Comment:
Seems like `path` could be `null` in the case here where `kind == PathKind.Empty`

Reply:
See assertion I added above. In this branch, we know that `GetPathKind(path) == kind == PathKind.RelativeToCurrentDirectory`

---
In reply to: [378078393](https://github.com/dotnet/roslyn/pull/41063#discussion_r378078393) [](ancestors = 378078393)